App.jsx

```diff
@@ -22,24 +20,27 @@ function App() {
   // Refresh collection when app comes back into foreground
   const handleAppStateChange = useCallback(nextAppState => {
     if (nextAppState === 'active' && !!sb?.currentUser) {
+      sb.setForegroundState();
       dispatch(refreshCollection());
+    } else if (nextAppState === 'background') {
+      sb?.setBackgroundState();
```

Comment:
@bang9 I think we should leave this because I will be introduce a complete sample later that includes a websocket based connection for Unified Inbox. If you think there is a concern, I can remove it for now.

Reply:
There are no concerns! I left a comment because I was thinking about what might be clearer from the customer's perspective and whether it is necessary. Since it won't affect the SDK's operation, it should be okay to leave it as it is